portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Listing/Listing.tsx

```diff
@@ -0,0 +1,1115 @@
+             */
+            const policy = policiesList[rowIndex];
+            /**
+             * Gateways which are deployed by any policy. These cannot be deployed again.
+             */
+            const allDepoloyedGateways = getAllDepoloyedGateways();
+            /**
+             * Gateways except the ones which are already deployed.
+             */
+            const gatewaysWithoutAppliedLabels = 
+                gatewayList.filter((item) => !policy.appliedGatewayLabels.includes(item));
+            const deployableGateways = 
+                gatewaysWithoutAppliedLabels.filter((item) => !allDepoloyedGateways.includes(item));
+
+            const handleSelectChange = (id: string) => (
+                event: React.ChangeEvent<{}>,
+                value: string[],
+            ) => {
+                const newSelectedGateways = selectedGateways.map((data) => {
+                    if (data.id === id) {
+                        return {
+                            ...data,
+                            gatewayLabels: value,
+                        };
+                    }
+                    return data;
+                });
+                setSelectedGateways(newSelectedGateways);
+            };
+
+            return ( 
+                <TableRow>
+                    <TableCell colSpan={1}/> 
+                    <TableCell colSpan={3}>      
+                        <Grid container spacing={2}>
+                            <Grid item xs={12} sm={6} md={4}>
+                                <Grid container spacing={2} alignItems='center'>
+                                    <Grid item xs={11}>
+                                        <Autocomplete
+                                            multiple
+                                            id='multi-select'
+                                            options={
+                                                deployableGateways.slice().sort((a, b) => 
+                                                    a.localeCompare(b, undefined, { sensitivity: 'base' }))}
+                                            value={getSelectedGatewayLabelsById(policy.id)}
+                                            onChange={handleSelectChange(policy.id)}
+                                            renderInput={(params) => (
+                                                <TextField
+                                                    // Prop spreading is forbidden in eslint, However, it is used here
+                                                    // as MUI AutoComplete component requires Prop spreading
+                                                    {...params}
+                                                    size='small'
+                                                    variant='outlined'
+                                                    label={intl.formatMessage({
+                                                        id: 'Select.Gateways',
```

Comment:
Let's use different ids for label and placeholder

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)